Habitica/src/main/java/com/habitrpg/android/habitica/ui/adapter/inventory/StableRecyclerAdapter.kt

```diff
@@ -25,6 +26,9 @@ class StableRecyclerAdapter : androidx.recyclerview.widget.RecyclerView.Adapter<
     var itemType: String? = null
     var context: Context? = null
     var activity: MainActivity? = null
+
+    var shopSpriteSuffix: String = ""

```

Comment:
This is never set to anything, right? Or am I overlooking something?

Reply:
Correct, this is just to keep it consistent with what was happening for shop headers, but it is not being used in this case